tests/admin/import/test-class-import-aioseo.php

```diff
@@ -24,68 +24,132 @@ public function setUp() {
 	}
 
 	/**
+	 * @covers WPSEO_Import_AIOSEO::run_detect
 	 * @covers WPSEO_Import_AIOSEO::detect
-	 * @covers WPSEO_Import_AIOSEO::detect_helper
 	 */
 	public function test_detect_with_data() {
 		$this->setup_post();
-		$this->assertEquals( $this->status( 'detect', true ), $this->class_instance->detect() );
+		$this->assertEquals( $this->status( 'detect', true ), $this->class_instance->run_detect() );
 	}
 
 	/**
-	 * @covers WPSEO_Import_AIOSEO::import
+	 * @covers WPSEO_Import_AIOSEO::run_import
 	 */
 	public function test_import_without_data() {
-		$result = $this->class_instance->import();
+		$result = $this->class_instance->run_import();
 		$this->assertEquals( $this->status( 'import', false ), $result );
 	}
 
 	/**
+	 * @covers WPSEO_Import_AIOSEO::run_import
 	 * @covers WPSEO_Import_AIOSEO::import
-	 * @covers WPSEO_Import_AIOSEO::import_metas
+	 * @covers WPSEO_Import_AIOSEO::import_opengraph
+	 * @covers WPSEO_Import_AIOSEO::import_post_opengraph
+	 * @covers WPSEO_Import_AIOSEO::meta_key_clone
+	 * @covers WPSEO_Import_AIOSEO::meta_keys_clone
 	 */
 	public function test_import_with_data() {
 		$post_id = $this->setup_post();
-		$result  = $this->class_instance->import();
+		$result  = $this->class_instance->run_import();
+
+		$seo_title       = get_post_meta( $post_id, WPSEO_Meta::$meta_prefix . 'title', true );
+		$seo_desc        = get_post_meta( $post_id, WPSEO_Meta::$meta_prefix . 'metadesc', true );
+		$robots_noindex  = get_post_meta( $post_id, WPSEO_Meta::$meta_prefix . 'meta-robots-noindex', true );
+		$robots_nofollow = get_post_meta( $post_id, WPSEO_Meta::$meta_prefix . 'meta-robots-nofollow', true );
+		$opengraph_image = get_post_meta( $post_id, WPSEO_Meta::$meta_prefix . 'opengraph-image', true );
+		$opengraph_title = get_post_meta( $post_id, WPSEO_Meta::$meta_prefix . 'opengraph-title', true );
+
+		$this->assertEquals( 'Test title', $seo_title );
+		$this->assertEquals( 'Test description', $seo_desc );
+		$this->assertEquals( 1, $robots_noindex );
+		$this->assertEquals( 1, $robots_nofollow );
+		$this->assertEquals( 'http://local.wordpress.test/wp-content/uploads/2018/01/actionable-seo.png', $opengraph_image );
+		$this->assertEquals( 'OpenGraph AIOSEO title', $opengraph_title );
+		$this->assertEquals( $this->status( 'import', true ), $result );
+	}
 
-		$seo_title = get_post_meta( $post_id, WPSEO_Meta::$meta_prefix . 'title', true );
-		$seo_desc  = get_post_meta( $post_id, WPSEO_Meta::$meta_prefix . 'metadesc', true );
+	/**
+	 * @covers WPSEO_Import_AIOSEO::__construct
+	 * @covers WPSEO_Import_AIOSEO::import
+	 * @covers WPSEO_Import_AIOSEO::meta_key_clone
+	 * @covers WPSEO_Import_AIOSEO::run_import
+	 * @covers WPSEO_Import_AIOSEO::set_missing_db_rights_status
+	 */
+	public function test_import_without_rights_to_temp_table() {
+		$mock = $this->getMockBuilder( 'wpdb' )
+					 ->setConstructorArgs( array( DB_USER, DB_PASSWORD, DB_NAME, DB_HOST ) )
```

Comment:
These lines should be indented one tab from `$mock`, not whatever indentation is used here.

Reply:
That's GitHub, unfortunately, they are indented correctly in the code.

Reply:
Not really - current indentation in the snippet above is three tabs + one space.